opsagent/opsagent.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script installs the Google Cloud Ops Agent on each node in the cluster.
+# See https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#default
+# for built-in configuration of Ops Agent.
+
+# Detect dataproc image version from its various names
+if (! test -v DATAPROC_IMAGE_VERSION) && test -v DATAPROC_VERSION; then
+  DATAPROC_IMAGE_VERSION="${DATAPROC_VERSION}"
+fi
```

Comment:
[Optional] Not going to block the PR on this, but this is just `DATAPROC_IMAGE_VERSION="${DATAPROC_IMAGE_VERSION:-${DATAPROC_VERSION}}"` (see https://tldp.org/LDP/abs/html/parameter-substitution.html and https://tldp.org/LDP/abs/html/refcards.html#AEN22728). Also in the other script.